internal/services/synapse/synapse_managed_private_endpoint_resource_test.go

```diff
@@ -61,7 +61,7 @@ func (r SynapseManagedPrivateEndpointResource) Exists(ctx context.Context, clien
 		if utils.ResponseWasNotFound(resp.Response) {
 			return utils.Bool(false), nil
 		}
-		return nil, fmt.Errorf("retrieving Synapse Managed Private Endpoints (Workspace %q / Resource Group %q): %+v", id.WorkspaceName, id.ResourceGroup, err)
+		return nil, fmt.Errorf("retrieving %s: %+v", id, err)
```

Comment:
this is a parsed ID, so it'll be returned as a pointer - as such this needs to be:

```suggestion
		return nil, fmt.Errorf("retrieving %s: %+v", *id, err)
```